src/nupic/encoders/multi.py

```diff
@@ -20,21 +20,21 @@
 # ----------------------------------------------------------------------
 
 from nupic.encoders.base import Encoder
-from nupic.encoders.scalar import ScalarEncoder
-from nupic.encoders.adaptive_scalar import AdaptiveScalarEncoder
-from nupic.encoders.date import DateEncoder
-from nupic.encoders.logarithm import LogEncoder
-from nupic.encoders.category import CategoryEncoder
-from nupic.encoders.sdr_category import SDRCategoryEncoder
-from nupic.encoders.delta import DeltaEncoder
-from nupic.encoders.scalar_space import ScalarSpaceEncoder
-from nupic.encoders.pass_through import PassThroughEncoder
-from nupic.encoders.sparse_pass_through import SparsePassThroughEncoder
-from nupic.encoders.coordinate import CoordinateEncoder
-from nupic.encoders.geospatial_coordinate import GeospatialCoordinateEncoder
+from nupic.encoders import ScalarEncoder
+from nupic.encoders import AdaptiveScalarEncoder
+from nupic.encoders import DateEncoder
+from nupic.encoders import LogEncoder
+from nupic.encoders import CategoryEncoder
+from nupic.encoders import SDRCategoryEncoder
+from nupic.encoders import DeltaEncoder
+from nupic.encoders import ScalarSpaceEncoder
+from nupic.encoders import PassThroughEncoder
+from nupic.encoders import SparsePassThroughEncoder
+from nupic.encoders import CoordinateEncoder
+from nupic.encoders import GeospatialCoordinateEncoder
```

Comment:
According to our [style guide](http://nupic.docs.numenta.org/prerelease/contributing/python-style-guide.html#imports), please import all encoders enclosed in parentheses to allow multiple lines.